docker/k8s/scripts/enclave_manager.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+enclave_manager="${TCF_HOME}/examples/enclave_manager/tcf_enclave_manager/enclave_manager.py"
+
+source ${TCF_HOME}/tools/build/_dev/bin/activate
+
+echo "starting enclave manager ..."
+python3 $enclave_manager --config-dir ${TCF_HOME}/docker/k8s
```

Comment:
Any specific reason why different config directory is used for kubernetes ?

Reply:
It's to address the problem of lacking way to customize the LMDB size before mering PR #45

Reply:
Now that PR #45 is now merged, so a different config directory is not needed and I see you removed  `--config-dir ${TCF_HOME}/docker/k8s`